subprojects/errorprone/errorprone.gradle

```diff
@@ -0,0 +1,20 @@
+description = "ErrorProne plugins for Mockito"
+
+apply from: "$rootDir/gradle/java-library.gradle"
+apply from: "$rootDir/gradle/dependencies.gradle"
+
+dependencies {
+    compile project.rootProject
+    compile libraries.errorprone
+
+    testCompile libraries.junit4
+    testCompile libraries.errorproneTestApi
+}
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
```

Comment:
Shall we keep a lower target, to allow benefit for older platforms ?

Reply:
ErrorProne must run on Java 8: https://errorprone.info/docs/installation

Reply:
OK that makes sense ;)